xcommon.cmake

```diff
@@ -402,6 +402,10 @@ function(XMOS_REGISTER_APP)
         message(FATAL_ERROR "APP_HW_TARGET not set in application Cmakelists")
     endif()
 
+    if(APP_MULTITILE_MERGE GREATER 2)
```

Comment:
Merging of only two tiles is supported so far, but potentially this could be indefinite with some loops to create more per-tile targets and do the xobjdump replace command for every tile.